apolloschurchapp/src/tabs/index.js

```diff
@@ -111,20 +104,45 @@ const HomeTab = createFeatureFeedTab({
 });
 
 const ReadTab = createFeatureFeedTab({
+  options: {
+    headerLeft: HeaderLeft,
+  },
   tabName: 'Read',
   feedName: 'READ',
 });
 
 const WatchTab = createFeatureFeedTab({
+  options: {
+    headerLeft: HeaderLeft,
+  },
   tabName: 'Watch',
   feedName: 'WATCH',
 });
 
 const PrayTab = createFeatureFeedTab({
+  options: {
+    headerLeft: HeaderLeft,
+  },
   tabName: 'Pray',
   feedName: 'PRAY',
 });
 
+const CustomConnectScreen = () => (
+  <ConnectScreenConnected ActionTable={ActionTable} ActionBar={ActionBar} />
+);
+
+const ConnectTabStack = createNativeStackNavigator();
+const ConnectTabStackNavigator = () => (
+  <ConnectTabStack.Navigator
+    options={{
+      headerLargeTitle: true,
```

Comment:
```suggestion
      headerHideShadow: true,
      headerLargeTitle: true,
```

Reply:
It actually is on the `ui-connected` component. there is a chance we don't need it since it's a different color but having this for now matches

Reply:
Never mind I was confused. Point is we are actually hiding shadows. Maybe we don't need to be and it would look ok but that can be for another PR.